ui/README.md

```diff
@@ -1,34 +1,92 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Console UI
+
+This part of the project contains the user interface for the Kafka console, developed using Next.js and the [PatternFly](https://patternfly.org) UI library.
 
 ## Getting Started
 
-First, run the development server:
+Create a `.env` file containing the details about where to find the API server, and some additional config.
+
+```.dotenv
+BACKEND_URL=http://localhost:8080
```

Comment:
This content looks duplicated, keeping the documentation up to date requires maintenance and we wanna minimize it.
Either you keep these information in one place or you put in place automation to keep the text synced.